lib/addons/p5.dom.js

```diff
@@ -1837,6 +1837,70 @@
    * @param {Number}            [val] volume between 0.0 and 1.0
    * @return {Number|p5.MediaElement} current volume or p5.MediaElement
    * @method volume
+   * @example
+	 * <div>
+	 * <code>
+	 * var audio;
+	 * var colorLow;
```

Comment:
to simplify, we can remove these color variables and just call background directly with the rgb values in draw